listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
```

Comment:
This looks a little odd to me - If we have two calls to the same function close together my preference is to save it in a variable:
```python
artist_mbids = row.get('artist_mbids')
if artist_mbids:
    for mbid in artist_mbids.split(','):
```
However I think we can take a shortcut here - we can have a default value of `""`, which when split will be an empty array:
```python
for mbid in row.get('artist_mbids', '').split(','):
```
But looking further, if `artist_mbids` isn't in the returned data, this field will be an empty array, so why can't we do this?:
```python
artist_mbids = row.get('artist_mbids', '').split(',')
``` 

and the same with `tags`